kubtest/charts/kubtest/values.yaml

```diff
@@ -143,6 +143,22 @@ cypress-executor:
   mongoDSN: "mongodb://kubtest-mongodb:27017"
   apiServerURI: "http://kubtest-api-server:8080"
 
+curl-executor:
+  enabled: true
+  nameOverride: "curl-executor"
+  fullnameOverride: "kubtest-curl-executor"
+  image:
+    repository: kubeshop/kubtest-curl-executor
+    pullPolicy: Always
+    # Overrides the image tag whose default is the chart appVersion.
+    # TODO we should stick to static version
+    tag: "latest"
+  service:
+    type: NodePort # for working with real k8s we should use "ClusterIP" type.
+    port: 8086
+  mongoDSN: "mongodb://kubtest-mongodb:27017"
```

Comment:
is it using mongo ?

Reply:
Right now it is with the in memory storage, but I want to make it to use mongo db if we are going to make it public.

Reply:
doesn't the api-server write the test results returned by executors to mongo? does the executor need to communicate directly with mongo also?

Reply:
Executors use also storage(mongo) when it receives an execution request, executor saves the execution to the storage with status "queued", then the worker is processing it, it changes status to "pending" and after the test is done it updates the status.

Reply:
thanks @nicufk - perhaps this is something we should provide an abstraction for at some point - so we don't depend on mongo for this in each executor...